salt/templates/virt/libvirt_domain.jinja

```diff
@@ -7,6 +7,9 @@
                 {% for dev in boot_dev %}
                 <boot dev='{{ dev }}' />
                 {% endfor %}
+		{% if loader %}
+		<loader readonly='yes' type='pflash'>{{ loader }}</loader>
```

Comment:
``readonly`` should be configurable and allow a ``no`` value.
``type`` should be configurable too to allow the ``rom`` value
While at it, could you add a ``secure`` flag (value yes or no) as documented in https://libvirt.org/formatdomain.html#elementsOS
Add the whole loader section in the template should go in it own commit since that can also be used for Xen or QEMU guests.

Reply:
Ack, changed the loader argument accordingly.